<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qgis_dev</class>
 <widget class="QMainWindow" name="qgis_dev">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qgis_dev</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>682</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuVector">
    <property name="title">
     <string>Vector</string>
    </property>
    <addaction name="actionAdd_Vector"/>
    <addaction name="actionNew_Vector"/>
   </widget>
   <widget class="QMenu" name="menuRaster">
    <property name="title">
     <string>Raster</string>
    </property>
    <addaction name="actionAdd_Raster"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuVector"/>
   <addaction name="menuRaster"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_Vector">
   <property name="text">
    <string>Add Vector</string>
   </property>
  </action>
  <action name="actionAdd_Raster">
   <property name="text">
    <string>Add Raster</string>
   </property>
  </action>
  <action name="actionNew_Vector">
   <property name="text">
    <string>New Vector</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
